packages/flutter_tools/lib/src/convert.dart

```diff
@@ -52,7 +52,7 @@ class Utf8Decoder extends cnv.Utf8Decoder {
     // was malformed.
     if (reportErrors && result.contains('\u{FFFD}')) {
       throwToolExit(
-        'Bad UTF-8 encoding found while decoding string: $result. '
+        'Bad UTF-8 encoding(U+FFFD; REPLACEMENT CHARACTER) found while decoding string: $result. '
```

Comment:
Can you add a space before your parantheses, like:
`        'Bad UTF-8 encoding (U+FFFD; REPLACEMENT CHARACTER) found while decoding string: $result. '`